packages/charting/controller/src/index.js

```diff
@@ -267,16 +267,18 @@ export const validate = (model = {}) => {
   const correctAnswerErrors = {};
   const categoryErrors = {};
 
+  const isLabelEmpty = (label) => label === '' || label === '<div></div>';
+
   categories.forEach((category, index) => {
     const { label } = category;
 
-    if (label === '' || label === '<div></div>') {
+    if (isLabelEmpty(label)) {
       categoryErrors[index] = 'Content should not be empty.';
     } else {
       const identicalAnswer = categories.slice(index + 1).some((c) => c.label === label);
 
       if (identicalAnswer) {
-        categoryErrors[index + 1] = 'Content should be unique.';
+        categoryErrors[index + 1] = categoryErrors[index] = 'Category names should be unique.';
```

Comment:
Can you check with @PatriciaRomaniuc if this shouldn't use translations?

Reply:
Validation is on the authoring side, so no need for translations. Translations are only for the player side.